src/libs/ui/src/organisms/LoginForm/index.tsx

```diff
@@ -0,0 +1,130 @@
+        {/* Email Input */}
+        <div className="inline-grid">
+          <div className='flex text-xs' >
+            <Label className="text-xs font-HeroNewUltraLight">Email</Label>
+
+            <Label className="ml-0 text-sm font-heroNewLight,font-sans text-red-600"> *</Label>
+          </div>
+
+          <Controller
+            render={({ field }) =>
+              <InputField
+                className={`rounded-none pt-1 pb-1 pl-4 pr-4 mt-3 mb-6 h-[48px] text-base border-[1px] ${errors[LoginFormEnum.Email] ? 'border-[#595959]' : 'border-[#d6d6d6]'
+                } ${isSubmitted && errors[LoginFormEnum.Email] ? 'focus:outline-blue-700' : 'focus:outline-none'}`}
+                type="email"
+                placeholder="Email *" {...field}
+              />}
+            control={control}
+            {...register(LoginFormEnum.Email, { required: LoginForm.Message })}
+
+          />
+        </div>
+        {errors[LoginFormEnum.Email] && (
+          <span style={{ color: '#ce4635' }} className="text-normal font-HeroNewBold mb-3">
+            {errors[LoginFormEnum.Email]?.message}
+          </span>
+        )}
+
+
+        {/* Password Input with Visibility Toggle */}
+        <div className="inline-grid">
+          <div className='flex text-xs'>
+            <Label className="text-xs text-black font-HeroNewUltraLight">Password</Label>
+
+            <Label className="text-sm font-heroNewLight,font-sans text-red-600"> *</Label>
+          </div>
+          <Controller
+            render={({ field }) =>
+              <PasswordFeild
+                className={`rounded-none mt-3 mb-[13px]  pt-1 pb-1 pl-4 pr-4 h-[48px] text-base border-[1px] ${errors[LoginFormEnum.Email] ? 'border-[#595959]' : 'border-[#d6d6d6]'
+                } ${isSubmitted && errors[LoginFormEnum.Email] ? 'focus:outline-blue-700' : 'focus:outline-none'}`}
+                type={showPassword ? 'text' : 'password'} // Password visibility toggle
+                placeholder="Password *"
+                {...field}
+                onChange={(e: { target: { value: string; }; }) => {
+                  field.onChange(e);
+                  setIsPasswordFieldEmpty(e.target.value === ''); // Check if the field is empty
+                }}
+                suffix={(
+                  !isPasswordFieldEmpty && (
+                    <button
+                      type="button"
+                      onClick={togglePasswordVisibility}
+                      className="text-gray-600 focus:outline-none"
+                    >
+                      {showPassword ? 'Hide' : 'Show'}
+                    </button>
+                  )
+                )}
+              />}
+            control={control}
+            {...register(LoginFormEnum.Password, { required: LoginForm.Message })}
+          />
+
+          {errors.password && (
+            <span style={{ color: '#ce4635' }} className="text-normal font-HeroNewBold   mt-3">
```

Comment:
**style={{ color: '#ce4635' }}** don't use colors like this, we have to declare in global and use variable name here